src/content/fr/2019/fonts.md

```diff
@@ -0,0 +1,682 @@
+featured_quote: Web fonts enable beautiful and functional typography on the web. Using web fonts not only empowers design, but it democratizes a subset of design, as it allows easier access to those who might not have particularly strong design skills. However, for all the good they can do, web fonts can also do great harm to your site's performance if they are not loaded properly.
+featured_stat_1: 74.9%
+featured_stat_label_1: 3P Font Requests that use Google Fonts
+featured_stat_2: 29%
+featured_stat_label_2: Percent of pages that include a Google Fonts stylesheet link
+featured_stat_3: 718
+featured_stat_label_3: Most font requests on a single page
+---
+
+## Introduction
+
+Les polices Web permettent une typographie belle et fonctionnelle sur le web. L'utilisation des polices Web permet non seulement de renforcer le design, mais aussi de démocratiser un sous-ensemble du design, car elle permet un accès plus facile à ceux qui n'ont pas forcément de solides compétences en matière de design. Cependant, malgré tout le bien qu'elles peuvent faire, les polices web peuvent également nuire considérablement aux performances de votre site, si elles ne sont pas chargées correctement.
+
+Sont-elles un atout pour le web ? Offrent-elles plus d'avantages que d'inconvénients ? Les sentiers des standards du web sont-ils suffisamment pavés pour encourager les meilleures pratiques de chargement des polices web par défaut ? Et si ce n'est pas le cas, que faut-il changer ? Voyons si nous pouvons répondre à ces questions en examinant comment les polices de caractères sont utilisées sur le web aujourd'hui.
+
+## Où avez-vous obtenu ces polices de caractères pour le web ?
+
+La première et la plus importante question : la performance. Un chapitre entier est consacré à la [performance](./performance), mais nous allons ici approfondir un peu les questions de performance spécifiques aux polices.
+
+L'utilisation de polices web hébergées permet une mise en œuvre et une maintenance faciles, mais l'auto-hébergement offre les meilleures performances. Étant donné que les polices web par défaut rendent le texte invisible pendant leur chargement (phénomène également connu sous le nom de <a hreflang="en" href="https://css-tricks.com/fout-foit-foft/">Flash of Invisible Text</a>, ou FOIT), les performances des polices web peuvent être plus critiques que celles des éléments non bloquants comme les images.
```

Comment:
`a hreflang="en"`/`a lang="en" hreflang="en"` (pense à wrapper les anglicismes avec des `lang="en"` :)

Reply:
Updated in https://github.com/HTTPArchive/almanac.httparchive.org/commit/6df484a83f05c285020d15faa5a0207a42c519e7